.eslintrc

```diff
@@ -24,7 +24,8 @@
           "__PLATFORM__",
           "__REPORT_REACT_DEVTOOLS_PORT__",
           "__FETCH_SUPPORT__",
-          "__NETWORK_INSPECT__"
+          "__NETWORK_INSPECT__",
+          "__APOLLO_DEVTOOLS_SHOULD_DISPLAY_PANEL__"
```

Comment:
Apollo dev tools are not displayed in the chrome dev tools by default.
It tries to check if window.apolloClient is set. Because we don't have apollo client in window (it is in web worker), I've created a separate bool for it.